tools/rpm2img

```diff
@@ -187,7 +188,7 @@ dd if="${PRIVATE_IMAGE}" of="${DISK_IMAGE}" conv=notrunc bs=1M seek=2005
 # BOTTLEROCKET-DATA
 truncate -s 1G "${DATA_IMAGE}"
 sgdisk --clear \
-   -n 0:1M:1023M -c 0:"BOTTLEROCKET-DATA" -t 0:8300 \
+   -n 0:1M:1023M -c 0:"BOTTLEROCKET-DATA" -t 0:"${BOTTLEROCKET_DATA_TYPECODE}" \
```

Comment:
Any particular reason why we don't want to use the standard "Linux data" partition label?

Reply:
I wanted to leave that for other disks that might show up, like EBS volumes.